### DISCUSSION TITLE
Can't  I load systemwide plugins like the in-built plugins?

### DISCUSSION THREAD

**Discussion by HubKing** [ohmyzsh/ohmyzsh#12517] [Help]
In-built plugins can be specified by a short name. 

    plugins=(git fzf)

But if I want things like auto-suggestion, completion, it seems that I have to use full path

    source /usr/share/zsh/plugins/....

It seems that systemwide plugins also follow the same structure and they are all in the same specific location. Can't I load them also by their shortnames like `zsh-autosuggestions` instead of `/usr/share/zsh/plugins/zsh-autosuggestions/zsh-autosuggestions.zsh`?

**Comment by mcornella** [comment]
By design Oh My Zsh only sources plugins that live within its directory or in the custom folder under `$ZSH_CUSTOM/plugins`.

In the end, a plugin is just a script file that can be sourced, so IMO it is more straightforward to use the `source /usr/share/zsh/plugins` method. But you can also create a symlink in the `$ZSH_CUSTOM/plugins` folder that points to the systemwide folder. Check out https://github.com/ohmyzsh/ohmyzsh/wiki/Customization for more info about the custom folder.
